ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
+          pip:
+            - pydantic<2
+            - google-api-python-client
+            - pillow
+            - diffusers==0.7.2
+            - transformers==4.24.0
+            - flax
+            - ml_dtypes==0.2.0
+            - jax[tpu]==0.4.11
+            - -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+            - fastapi
+            - pillow
+            - ray[serve,default]==2.9.0
+  rayClusterConfig:
+    rayVersion: '2.9.0' # Should match the Ray version in the image of the containers
+    ######################headGroupSpecs#################################
+    # Ray head pod template.
+    headGroupSpec:
+      # The `rayStartParams` are used to configure the `ray start` command.
+      # See https://github.com/ray-project/kuberay/blob/master/docs/guidance/rayStartParams.md for the default settings of `rayStartParams` in KubeRay.
+      # See https://docs.ray.io/en/latest/cluster/cli.html#ray-start for all available options in `rayStartParams`.
+      rayStartParams:
+        dashboard-host: '0.0.0.0'
+      # Pod template
+      template:
+        spec:
+          containers:
+          - name: ray-head
+            image: rayproject/ray:2.9.0-py310
+            ports:
+            - containerPort: 6379
+              name: gcs
+            - containerPort: 8265
+              name: dashboard
+            - containerPort: 10001
+              name: client
+            - containerPort: 8000
+              name: serve
+            volumeMounts:
+              - mountPath: /tmp/ray
+                name: ray-logs
+            resources:
+              limits:
+                cpu: "2"
+                memory: "8G"
+              requests:
+                cpu: "2"
+                memory: "8G"
+          volumes:
+            - name: ray-logs
+              emptyDir: {}
+    workerGroupSpecs:
+    # The pod replicas in this group typed worker
+    - replicas: 1
+      minReplicas: 1
+      maxReplicas: 10
+      numOfHosts: 2
+      groupName: tpu-group
+      rayStartParams: {}
+      # Pod template
+      template:
+        spec:
+          containers:
+          - name: ray-worker
+            image: rayproject/ray:2.9.0-py310
+            resources:
+              limits:
+                cpu: "1"
+                ephemeral-storage: 20Gi
+                google.com/tpu: "4"
+                memory: 80G
+              requests:
+                cpu: "1"
+                ephemeral-storage: 20Gi
+                google.com/tpu: "4"
+                memory: 80G
+          nodeSelector:
+            cloud.google.com/gke-tpu-accelerator: tpu-v4-podslice
+            cloud.google.com/gke-tpu-topology: 2x2x2
+        metadata:
+          labels:
```

Comment:
Why do we need these labels?

Reply:
I don't think these are needed actually, I'll remove them.